src/features/customerView/CustomerView.tsx

```diff
@@ -86,7 +86,24 @@ const CustomerView = ({
             />
           ))}
 
-        <OpenInvoicesCard
+        <InvoiceCard
+          berthLeases={berthLeases}
+          className={styles.fullWidth}
+          customer={{
+            id: customerProfile.customerId as string,
+            email: customerProfile.primaryEmail,
+          }}
+          invoiceData={invoices}
+          refetchQueries={refetchQueries}
+        />
+
+        {/* TODO: 
```

Comment:
What about this TODO?

Reply:
I'll leave it here for now, and I will make sure to remove it while implementing the next task in the story